android/BOINC/app/src/main/java/edu/berkeley/boinc/client/Monitor.kt

```diff
@@ -247,8 +321,8 @@ class Monitor : LifecycleService() {
             val actionCode = intent.getIntExtra("action", -1)
             if (Logging.DEBUG) Log.d(Logging.TAG, "Monitor.onStartCommand() with action code: $actionCode")
             when (actionCode) {
-                1 -> lifecycleScope.launch { setClientRunMode(RUN_MODE_NEVER) }
```

Comment:
Why remove the space?

Reply:
Probably done automatically and why a space.

Reply:
I can changed it, but am sure Android Studio will remove them again.